docker/README.md

```diff
@@ -0,0 +1,46 @@
+## docker installation
+
+1. build the tradenote image with your custom parameters
+```
+docker build . -t tradenote --build-arg PARSE_INIT=APPLICATION_ID --build-arg PARSE_URL=http://pi:1337/parse
+```
+
+2. run mongodb, parse server tradenote via docker compose file
+```
+docker-compose up -d
+```
+
+3. modify PARSE_INIT, PARSE_MASTER_KEY, PARSEHOST in addclass.sh to match your environment, run addclass.sh to add classes
+```
+chmod +x ./addclass.sh
+./addclass
+```
+
+4. register user via http://server:7777/register
+
+5. modify parameters to match your environment in localdashboard.yaml, run a standalone parse-dashboard container on local computer
+```
+docker-compose -f localdashboard.yaml up -d
+```
+
+6. login to parse dashboard, add the following columns and data to "_User"
+```
+user - type Pointer
+avatar - type file
+accounts - type array, add your broker id accounts :
+[
+ {
+  "value": "XY000001","label": "Live Account"
+  },
+  {
+    "value": "XY000002","label": "Demo Account"
+  }
+]
+```
+
+7.clean browser session and cache, login tradenote http://server:7777
+
+8. repeat step 7 if login failed, once login success, you can remove the local parse-dashboard
```

Comment:
As stated earlier, I would make this step optional. I personnaly use the dashboard very often. It's practical to check things and remove things if needed.